cmake/Modules/LocalLibargparse.cmake

```diff
@@ -23,8 +23,6 @@ else()
         libargparse
         GIT_REPOSITORY "https://github.com/kmurray/libargparse.git"
         GIT_TAG ${AVIF_LIBARGPARSE_GIT_TAG}
-        # TODO(vrabaud) remove once CMake 3.13 is not supported anymore.
-        PATCH_COMMAND git apply --ignore-whitespace "${AVIF_SOURCE_DIR}/ext/libargparse.patch"
```

Comment:
Yannis: Thanks for the review.

@vrabaud Vincent: Could you double check this and make sure I did this TODO correctly? Thanks.

Reply:
That is fine: the CI checks it.